packages/cli/src/services/project.service.ee.ts

```diff
@@ -299,7 +364,13 @@ export class ProjectService {
 		});
 	}
 
-	async addUser(projectId: string, userId: string, role: ProjectRole) {
+	/**
```

Comment:
I had to revert the changes to this method to fix the test, but the changes did make sense. I'll try to fix this and add additional tests next week.